src/stock_str.rs

```diff
@@ -275,6 +275,42 @@ pub enum StockMessage {
 
     #[strum(props(fallback = "Download maximum available until %1$s"))]
     DownloadAvailability = 100,
+
+    #[strum(props(fallback = "Inbox"))]
+    Inbox = 103,
+
+    #[strum(props(fallback = "Outbox"))]
+    Outbox = 104,
+
+    #[strum(props(fallback = "Storage on %1$s"))]
+    StorageOnDomain = 105,
+
+    #[strum(props(fallback = "One moment…"))]
+    OneMoment = 106,
+
+    #[strum(props(fallback = "Connected"))]
+    Connected = 107,
+
+    #[strum(props(fallback = "Connecting…"))]
+    Connecting = 108,
+
+    #[strum(props(fallback = "Updating…"))]
+    Updating = 109,
+
+    #[strum(props(fallback = "Sending…"))]
+    Sending = 110,
+
+    #[strum(props(fallback = "Your last message was sent successfully."))]
+    LastMsgSentSuccessfully = 111,
+
+    #[strum(props(fallback = "Error: %1$s"))]
+    Error = 112,
+
+    #[strum(props(fallback = "Your provider does not support reading quota information."))]
+    QuotaNotSupported = 113,
```

Comment:
original string was "Quota not supported by your provider." - but i think, this is a bit misleading, as provider may still have quota.

to the wording "quota": we avoid that in the title and also if things work out, many users seems to have no good idea about what "quota" means (see discussions in dc testing group). but it may be good to leave the word "quota" in the error message so that more experienced user get an idea what this is about.

Reply:
As we are already using the domain instead of "your provider" in `StorageOnDomain`, maybe use it here, too?

`"%1$s does not support reading quota information."`

Or: 

`"No quota information provided by %1$s."`

Reply:
tbh, i like the string saying "Your provider" more. also, not sure if it is important to repeat the domain name.

but, i would also be fine with your suggestions, if you insist :)

Reply:
The current wording is fine, too I think

Reply:
i am still a bit unhappy about the string. also i cannot really imagine how to translate that eg. to german, "Kontingentsinformationen"?! and translators who are not that close to the email land as we will probably come up with even weirder translations.

maybe strike "Quota" completely and just say "Not supported by your provider"?  that is also shorter, which is a good thing as this text will be very often shown, rather prominently. or is "Quota" fine? not sure.

suggestions are still very welcome :)

Reply:
> Not supported by your provider

:+1:

Reply:
"Unknown" does not really add any additional information, but, yet another idea: we could just leave the whole quota block away, which also means one string less to translate.

Reply:
> but, yet another idea: we could just leave the whole quota block away, which also means one string less to translate.

i also thought about that, however, that will look weird when it flashes up shortly on opening - and also, ppl may think there is a bug or so, when sometimes the section "Storage on xy" is shown and sometimes not.